site/content/docs/main/csi-snapshot-data-movement.md

```diff
@@ -0,0 +1,401 @@
+
+```
+velero install --use-node-agent
+```
+
+### Configure Node Agent DaemonSet spec
+
+After installation, some PaaS/CaaS platforms based on Kubernetes also require modifications the node-agent DaemonSet spec. 
+The steps in this section are only needed if you are installing on RancherOS, OpenShift, VMware Tanzu Kubernetes Grid 
+Integrated Edition (formerly VMware Enterprise PKS), or Microsoft Azure.  
+
+
+**RancherOS**
+
+
+Update the host path for volumes in the node-agent DaemonSet in the Velero namespace from `/var/lib/kubelet/pods` to 
+`/opt/rke/var/lib/kubelet/pods`.  
+
+```yaml
+hostPath:
+  path: /var/lib/kubelet/pods
+```
+
+to
+
+```yaml
+hostPath:
+  path: /opt/rke/var/lib/kubelet/pods
+```
+
+
+**OpenShift**
+
+
+To mount the correct hostpath to pods volumes, run the node-agent pod in `privileged` mode.
+
+1. Add the `velero` ServiceAccount to the `privileged` SCC:
+
+    ```
+    $ oc adm policy add-scc-to-user privileged -z velero -n velero
+    ```
+
+2. Modify the DaemonSet yaml to request a privileged mode:
+
+    ```diff
+    @@ -67,3 +67,5 @@ spec:
+                  value: /credentials/cloud
+                - name: VELERO_SCRATCH_DIR
+                  value: /scratch
+    +          securityContext:
+    +            privileged: true
+    ```
+
+    or
+
+    ```shell
+    oc patch ds/node-agent \
+      --namespace velero \
+      --type json \
+      -p '[{"op":"add","path":"/spec/template/spec/containers/0/securityContext","value": { "privileged": true}}]'
+    ```
+
+If node-agent is not running in a privileged mode, it will not be able to access snapshot volumes within the mounted 
+hostpath directory because of the default enforced SELinux mode configured in the host system level. You can 
+[create a custom SCC][6] to relax the 
+security in your cluster so that node-agent pods are allowed to use the hostPath volume plugin without granting 
+them access to the `privileged` SCC.  
+
+By default a userland openshift namespace will not schedule pods on all nodes in the cluster.  
+
+To schedule on all nodes the namespace needs an annotation:  
+
+```
+oc annotate namespace <velero namespace> openshift.io/node-selector=""
+```
+
+This should be done before velero installation.  
+
+Or the ds needs to be deleted and recreated:  
+
+```
+oc get ds node-agent -o yaml -n <velero namespace> > ds.yaml
+oc annotate namespace <velero namespace> openshift.io/node-selector=""
+oc create -n <velero namespace> -f ds.yaml
+```
+
+**VMware Tanzu Kubernetes Grid Integrated Edition (formerly VMware Enterprise PKS)**  
+
+You need to enable the `Allow Privileged` option in your plan configuration so that Velero is able to mount the hostpath.  
+
+The hostPath should be changed from `/var/lib/kubelet/pods` to `/var/vcap/data/kubelet/pods`
+
+```yaml
+hostPath:
+  path: /var/vcap/data/kubelet/pods
+```
+
+**Microsoft Azure**
+
+If you are using [Azure Files][7], you need to add `nouser_xattr` to your storage class's `mountOptions`. 
```

Comment:
This issue was already fixed by Restic several years ago: [3034](https://github.com/restic/restic/pull/3034)
I think we can safely remove this configuration.

Reply:
Looks like this xattr issue is Restic specific, Kopia doesn't handle xattr. 
Therefore, I've removed this for both fs backup and data mover.